cxf/soap-server/README.md

```
@@ -0,0 +1,100 @@
+3. Incoming SOAP messages are logged and a response is returned
+4. A built-in test caller fires once after 5 seconds to verify the server works
+
+## Running the Example
+
+```bash
+camel run *
+```
+
+You should see:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)